File: java/src/linkedlist/LC_237_DeleteNodeInALinkedList.java
package linkedlist;
// https://leetcode.com/problems/delete-node-in-a-linked-list


public  class LC_237_DeleteNodeInALinkedList{
    public void deleteNode(ListNode node) {
        node.val = node.next.val;
        node.next = node.next.next;
    }
} 

File: python/linkedlist/LC_237_DeleteNodeInALinkedList.py
# https://leetcode.com/problems/delete-node-in-a-linked-list

from typing import Optional
from linkedlist.listnode import ListNode


class Solution:
    def deleteNode(self, node: Optional[ListNode]):
        """
        :type node: ListNode
        :rtype: void Do not return anything, modify node in-place instead.
        """
        node.val = node.next.val
        node.next = node.next.next
